src/plugins/data/public/antlr/opensearch_ppl/grammar/OpenSearchPPLParser.g4

```diff
@@ -775,11 +775,11 @@ valueList
    ;
 
 qualifiedName
-   : ident (DOT ident)* # identsAsQualifiedName
+   : ID // (DOT ident)* # identsAsQualifiedName
    ;
 
 tableQualifiedName
-   : tableIdent (DOT ident)* # identsAsTableQualifiedName
+   : ID // tableIdent (DOT ident)* # identsAsTableQualifiedName
```

Comment:
do we not support those syntaxes?

Reply:
This Antlr file was used for parsing, and so it includes every grammatically correct possibility. But when we use it for code completion, a lot of the grammatically correct options all being displayed don't make sense. In this instance, I believe it was showing the table as a grammatically correct option directly after the table name + whitespace was already put in, which was happening since that's how the Antlr file was written for easier parsing.